test/mocks.dart

```diff
@@ -34,6 +35,9 @@ class MockPaymentMethodSelectorController extends Mock
 class MockCreditCardPaymentMethodController extends Mock
     implements CreditCardPaymentMethodController {}
 
+class MockPMobileBankingPaymentMethodSelectorController extends Mock
```

Comment:
Do you mean `MockMobileBankingPaymentMethodSelectorController`?
```suggestion
class MockMobileBankingPaymentMethodSelectorController extends Mock
```